tests/server/testServerWithStorage.js

```diff
@@ -562,39 +716,35 @@ sjrk.storyTelling.server.testServerWithStorageDefs.getSavedStory = function (sto
  * in the database
  *
  * @param {String} data - the data returned by the call
- * @param {Object} request - the Kettle request component
  * @param {Object} expectedStory - the expected story model
  * @param {Object} fileOptions - options related to the uploaded files
  * @param {Object} completionEvent - an event to fire on test completion
  * @param {Boolean} authoringEnabled - a flag indicating whether authoring is enabled
  */
-sjrk.storyTelling.server.testServerWithStorageDefs.verifyStoryPersistence = function (data, request, expectedStory, fileOptions, completionEvent, authoringEnabled) {
-    var binaryRenameMap = request.binaryRenameMap;
+sjrk.storyTelling.server.testServerWithStorageDefs.verifyStoryPersistence = function (data, expectedStory, fileOptions, completionEvent, authoringEnabled) {
     var parsedData = JSON.parse(data);
 
     if (authoringEnabled) {
         // update the expected model to use the
         // dynamically-generated file name before we
         // test on it
         var updatedModel = fluid.copy(expectedStory);
+
         if (fileOptions) {
-            updatedModel.content[0][fileOptions.urlProp] =
-                fileOptions.expectedUploadedFilesHandlerPath
-                + binaryRenameMap[testStoryModel.content[0][fileOptions.urlProp]];
+            var filePath = parsedData.content[0][fileOptions.urlProp];
+
+            updatedModel.content[0][fileOptions.urlProp] = filePath;
+
+            // verify that the file exists in the expected location
+            var exists = fs.existsSync(filePath);
+            jqUnit.assertTrue("Uploaded file exists: " + filePath, exists);
         }
 
         // Strip the _rev field from the parsedData
         parsedData = fluid.censorKeys(parsedData, "_rev");
 
         jqUnit.assertDeepEq("Saved story data is as expected", updatedModel, parsedData);
 
-        if (fileOptions) {
-            var exists = fs.existsSync(fileOptions.expectedUploadDirectory
-                + binaryRenameMap[testStoryModel.content[0][fileOptions.urlProp]]);
-
-            jqUnit.assertTrue("Uploaded file exists", exists);
-        }
-
         if (completionEvent && fileOptions) {
             completionEvent.fire(parsedData.content[0][fileOptions.urlProp]);
         }
```

Comment:
Shall the else block at line 751 to verify an error response when `authoringEnabled === false`?